src/citrine/informatics/design_spaces/product_design_space.py

```diff
@@ -26,41 +27,55 @@ class ProductDesignSpace(Resource['ProductDesignSpace'], DesignSpace):
 
     _response_key = None
 
-    uid = properties.Optional(properties.UUID, 'id', serializable=False)
-    name = properties.String('config.name')
-    description = properties.Optional(properties.String(), 'config.description')
-    dimensions = properties.List(properties.Object(Dimension), 'config.dimensions')
-    typ = properties.String('config.type', default='Univariate', deserializable=False)
-    status = properties.String('status', serializable=False)
-    status_info = properties.Optional(
-        properties.List(properties.String()),
-        'status_info',
-        serializable=False
-    )
-    archived = properties.Boolean('archived', default=False)
-    experimental = properties.Boolean("experimental", serializable=False, default=True)
-    experimental_reasons = properties.Optional(
-        properties.List(properties.String()),
-        'experimental_reasons',
-        serializable=False
+    subspaces = properties.List(properties.Union(
+        [properties.UUID, properties.Object(DesignSpace)]
+    ), 'config.subspaces', default=[])
+    dimensions = properties.Optional(
+        properties.List(properties.Object(Dimension)), 'config.dimensions'
     )
+    typ = properties.String('config.type', default='ProductDesignSpace', deserializable=False)
+    # Product design spaces may not be embedded in other subspaces, hence status is required
+    status = properties.String('status', serializable=False)
 
     # NOTE: These could go here or in _post_dump - it's unclear which is better right now
     module_type = properties.String('module_type', default='DESIGN_SPACE')
     schema_id = properties.UUID('schema_id', default=UUID('6c16d694-d015-42a7-b462-8ef299473c9a'))
 
-    def __init__(self,
+    def __init__(self, *,
                  name: str,
                  description: str,
-                 dimensions: List[Dimension],
+                 subspaces: Optional[List[Union[UUID, DesignSpace]]] = None,
+                 dimensions: Optional[List[Dimension]] = None,
                  session: Session = Session()):
         self.name: str = name
         self.description: str = description
-        self.dimensions: List[Dimension] = dimensions
+        self.subspaces: List[Union[UUID, DesignSpace]] = subspaces or []
+        self.dimensions: List[Dimension] = dimensions or []
         self.session: Session = session
 
+    def dump(self) -> dict:
```

Comment:
There's essentially a pre-dump step here. It might make sense to add `pre_dump` as a standard method, rather than completely overriding `dump()` as I do here.

Reply:
We could do that.  Non-blocking, though.